### PR TITLE
LRQA-73706 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-73706

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase [hunk 1]
```diff
@@ -2015,4 +2015,69 @@ definition {
 		}
 	}
 
+	@description = "This test asserts that the User's available views within the page shall reflect the permissions of the Role in the Selected Account."
+	@priority = "5"
+	test ThePageActionsAreConsistentWithViewRolesPermissionsInSelectedAccount {
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:2020]: @melodywu Please make sure the test title and description are still accurate with the new test case

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase [hunk 2]
```diff
@@ -2015,4 +2015,69 @@ definition {
...
+		task ("Given add account a new user and account role with view permission") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:2024]: What is this step for?

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
@melodywu Minor changes, please see comments. Maybe we can also expand this further and add another entry that a user shouldn't be able to view? What do you think? Thanks!

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase (hunk 1 above)
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase:2020]: Done

File: modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase (hunk 2 above)
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/AccountPermissions.testcase]: Done

**Comment by melodywu** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Just started reviewing :)

**Comment by patriciajeanperez** [comment]
@melodywu Please see changes in https://github.com/patriciajeanperez/liferay-portal/pull/689. Please make use of other JSON macros to simplify test steps. Let me know if you have any questions!

**Pull request closed without merging**